templates/home/home-en.html

```diff
@@ -1,12 +1,11 @@
 ---
-{
-	"altLangPage": "home-fr.html",
-	"dateModified": "2021-04-06",
-	"language": "en",
-	"layout": "layout-home",
-	"share": "true",
-	"title": "Home - Canada.ca"
-}
+title: Home - Canada.ca
+language: en
+altLangPage: home-fr.html
+dateModified: 2021-06-29
+layout: layout-home
+noReportProblem: true
+share: true
```

Comment:
Just a comment: Should not the "Share this page" be there by default? @Ricokola how it is currently setup in MWS?